_layouts/base_slides.html

```diff
@@ -25,6 +25,10 @@
 layout: true
 class: center, middle, inverse
 
+<div class="my-header"><span>
+<a href="{{ site.baseurl }}/topics/{{ topic.name }}" title="Return to topic page" ><i class="fa fa-level-up" aria-hidden="true"></i></a>
```

Comment:
I think a "home" icon would be more appropriate.

Reply:
With "home" I'd expect to go back to the website home, while this goes back to the topic.